Arc-i-Tech/src/main/java/com/arcitech/dto/LoginRequest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2024, Arc-i-Tech. All rights reserved.
+ * visit www.arc-i-tech.in
+ *
+ */
+package com.arcitech.dto;
+
+/**
+ * DTO for login request.
+ * 
+ * @author Ajay G
+ * 
+ */
+public class LoginRequest {
+	private String username;
+	private String password;
+
+	/**
+	 * @return the username
+	 */
+	public String getUsername() {
```

Comment:
we can Replace manually written getter and setter methods with Lombok's annotations to reduce boilerplate code.